### PR TITLE
Fix migrator

### PR DESCRIPTION
**Pull request by jayfurmanek**
closes #45 
Add builds for these new platforms.
Requires an explicit dependency on make.

Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jayfurmanek** [comment]
@conda-forge-admin, please rerender

**Comment by jayfurmanek** [comment]
The patch for ppc64le disables the linalg and eigen tests. Each have a subtest that fails due to slight accuracy differences. As there are no tolerance adjustments, we make these as fail.

**Comment by jayfurmanek** [comment]
@isuruf fyi

**Comment by isuruf** [comment]
Has this been reported upstream?

**Comment by jayfurmanek** [comment]
Not yet. I plan to and will repost back here once its open. I need to get a savannah account first. It looks like the same scenario on aarch64, so I applied the xfail patch there too due to lack of tolerance controls.

**Comment by jayfurmanek** [comment]
Savannah bug opened on GSL upstream:
https://savannah.gnu.org/bugs/index.php?56843

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,9 +10,10 @@ source:
   patches:
     # diff from source distribution to https://github.com/ampl/gsl/tree/v2.4.0
     - patch_cmake_for_windows.diff  # [win]
+    - xfails-on-ppc64le-for-tolerance.patch  # [not x86_64]
```
Comment (isuruf) [inline recipe/meta.yaml]: Can you add the savannah link as a comment above?

Comment (jayfurmanek) [reply recipe/meta.yaml]: surely

**Pull request merged**